services/appengine-standard-service/src/main/proto/cloud/trace/proto/trace_id.proto

```diff
@@ -15,7 +15,6 @@
 syntax = "proto3";
 
 option java_package = "com.google.protos.cloud.trace";
-option optimize_for = LITE_RUNTIME;
```

Comment:
Should there be a TODO to add this line back?

Reply:
I fixed it. The problem was that I forgot to update the protoc version.